Sources/SnapshotTesting/Snapshotting/URLRequest.swift

```diff
@@ -16,4 +16,39 @@ extension Snapshotting where Value == URLRequest, Format == String {
 
     return ([method] + headers + body).joined(separator: "\n")
   }
+  
+  /// A snapshot strategy for comparing requests based on a cURL representation.
+  public static let curl = SimplySnapshotting.lines.pullback { (request: URLRequest) in
+
+    var components = ["curl"]
+
+    // HTTP Method
+    let httpMethod = request.httpMethod!
+    switch httpMethod {
+    case "GET": break
+    case "HEAD": components.append("--head")
+    default: components.append("-X \(httpMethod)")
+    }
+
+    // Headers
+    if let headers = request.allHTTPHeaderFields {
+      for (field, value) in headers where field != "Cookie" {
```

Comment:
It seems that alamofire omits the `"Cookie"` header because it uses a `URLSessionConfiguration` under the hood. We may need to pluck out the cookie header and add it to the curl request via something like:

```swift
if let cookie = request.allHTTPHeaderFields["Cookie"] {
  components.append("-b \"\(cookie)\"")
}
```

I believe that is correct, but haven't thought too much about it myself.

Reply:
I had realised this but had also noticed [another script](https://gist.github.com/shaps80/ba6a1e2d477af0383e8f19b87f53661d) also skipping this (without taking the one from the Cookie field)

Wasn't sure whether there was something I didn't know about cURL/Cookies in headers so decided to skip over this behaviour.

Happy to add this, but will validate the functionality first